<ui version="4.0" >
 <class>importdialog</class>
 <widget class="QDialog" name="importdialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Import data - Veusz</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>&amp;Filename</string>
       </property>
       <property name="buddy" >
        <cstring>filenameedit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filenameedit" >
       <property name="toolTip" >
        <string>Enter the filename to be imported here</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browsebutton" >
       <property name="text" >
        <string>&amp;Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="methodtab" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="standardtab" >
      <attribute name="title" >
       <string>&amp;Standard</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>File preview:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="previewedit" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>4</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <family>Sans Serif</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="lineWrapMode" >
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>5</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>The descriptor describes the format of the data in the file. Each dataset is described separated with spaces. Commas separate the dataset names from the description of the errors

e.g.	z	[z with no errors - 1 column for dataset]
	x,+-	[x with symmetric errors - 2 columns for dataset]
	y,+,-	[y with asymmetric errors - 3 columns for dataset]
	x[1:5],+,-	[x_1 to x_5, each with asymmetric errors - 15 columns in total]
	x y,+-	[x with no errors, y with symmetric errors - 3 columns in total]
</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>&amp;Descriptor</string>
           </property>
           <property name="buddy" >
            <cstring>descriptoredit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="descriptoredit" >
           <property name="toolTip" >
            <string>Names of columns when importing data, e.g. "x y" or "a[:]"</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="blockcheckbox" >
         <property name="toolTip" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">If this is selected, blank lines or the word "no" are used to separate the file into blocks. An underscore followed by the block number is added to the dataset names&lt;/p>&lt;/body>&lt;/html></string>
         </property>
         <property name="text" >
          <string>Read data in bloc&amp;ks</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="csvtab" >
      <attribute name="title" >
       <string>CS&amp;V</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>File preview:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="previewtablecsv" >
         <property name="editTriggers" >
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Comma Separated Value (CSV) files are often used to export data from applications such as Excel and OpenOffice. Veusz can read data from these files. At the top of each column of data, a dataset name can be given for the data below. Multiple datasets can be placed below each other if new names are given.

To import error bars, columns with the names "+", "-" or "+-" should be given in columns immediately to the right of the dataset, for positive, negative or symmetric errors.

Veusz can also read data organised in rows rather than columns.
</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="1" >
          <widget class="QComboBox" name="directioncombo" >
           <property name="toolTip" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Are the data arranged in columns or rows?&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <item>
            <property name="text" >
             <string>Columns</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Rows</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>&amp;Direction</string>
           </property>
           <property name="buddy" >
            <cstring>directioncombo</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="prefixedit" >
           <property name="toolTip" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Optional text prepended to the name of each dataset&lt;/p>&lt;/body>&lt;/html></string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>&amp;Prefix</string>
           </property>
           <property name="buddy" >
            <cstring>prefixedit</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="fitstab" >
      <attribute name="title" >
       <string>FI&amp;TS</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="fitslabel" >
           <property name="text" >
            <string>List of HDUs in FITS file:</string>
           </property>
           <property name="wordWrap" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="fitsimportstatus" >
           <property name="text" >
            <string/>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="fitshdulist" >
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="4" column="0" >
          <widget class="QLabel" name="label_14" >
           <property name="text" >
            <string>Negative error FITS column:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="fitsdatasetname" >
           <property name="toolTip" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Column or image specified is imported using the name given here&lt;/p>&lt;/body>&lt;/html></string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_11" >
           <property name="text" >
            <string>Data FITS column:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="label_10" >
           <property name="text" >
            <string>Dataset name to use:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_12" >
           <property name="text" >
            <string>Symmetric error FITS column:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" >
          <widget class="QComboBox" name="fitsnegcolumn" >
           <property name="toolTip" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Column used for negative errors, if appropriate&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="editable" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="label_13" >
           <property name="text" >
            <string>Positive error FITS column:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QComboBox" name="fitssymcolumn" >
           <property name="toolTip" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Column used for symmetric errors, if appropriate&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="editable" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QComboBox" name="fitsposcolumn" >
           <property name="toolTip" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Column used for positive errors, if appropriate&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="editable" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QComboBox" name="fitsdatacolumn" >
           <property name="toolTip" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">If importing a table, this is the column used to provide the main data&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="editable" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2d" >
      <attribute name="title" >
       <string>&amp;2D</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>File preview:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="twod_previewedit" >
         <property name="lineWrapMode" >
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label_20" >
           <property name="text" >
            <string>Datasets:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="twod_datasetsedit" >
           <property name="toolTip" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">A space separated list of dataset names to import from the file&lt;/p>&lt;/body>&lt;/html></string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="1" column="2" >
          <widget class="QLabel" name="label_17" >
           <property name="text" >
            <string>to</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_18" >
           <property name="text" >
            <string>Range of &amp;Y:</string>
           </property>
           <property name="buddy" >
            <cstring>twod_yminedit</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="3" >
          <widget class="QLineEdit" name="twod_ymaxedit" >
           <property name="toolTip" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Enter a number for the maximum coordinate of the Y axis (default number of rows in file)&lt;/p>&lt;/body>&lt;/html></string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="twod_xminedit" >
           <property name="toolTip" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Enter a number for the minimum coordinate of the X axis (default 0)&lt;/p>&lt;/body>&lt;/html></string>
           </property>
          </widget>
         </item>
         <item row="0" column="3" >
          <widget class="QLineEdit" name="twod_xmaxedit" >
           <property name="toolTip" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Enter a value for a maximum coordinate of the X axis (default is number of columns in file)&lt;/p>&lt;/body>&lt;/html></string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="twod_yminedit" >
           <property name="toolTip" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Enter a number for the minimum coordinate of the Y axis (default 0)&lt;/p>&lt;/body>&lt;/html></string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="label_15" >
           <property name="text" >
            <string>Range of &amp;X:</string>
           </property>
           <property name="buddy" >
            <cstring>twod_xminedit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="2" >
          <widget class="QLabel" name="label_16" >
           <property name="text" >
            <string>to</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="twod_invertrowscheck" >
         <property name="text" >
          <string>Invert &amp;rows when reading</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="twod_invertcolscheck" >
         <property name="text" >
          <string>Invert colu&amp;mns when reading</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="twod_transposecheck" >
         <property name="text" >
          <string>Trans&amp;pose rows and columns</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_19" >
         <property name="text" >
          <string>Data are read as a 2D matrix from the file.  X is read from left to right, Y from bottom to top, by default. These parameters listed here can be altered by including xrange A B, yrange A B, invertrows, invertcols and transpose as lines in the data file. Multiple datasets can be included by separating with blank lines.
</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="linkcheckbox" >
     <property name="toolTip" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Data imported are not stored in the Veusz saved file, but are reloaded each time the Veusz file is opened, or Data-&amp;gt;Reload is selected on the menu.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="text" >
      <string>&amp;Link datasets to file</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="importbutton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Import</string>
       </property>
       <property name="shortcut" >
        <string>Alt+I</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closebutton" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>filenameedit</tabstop>
  <tabstop>browsebutton</tabstop>
  <tabstop>methodtab</tabstop>
  <tabstop>previewedit</tabstop>
  <tabstop>descriptoredit</tabstop>
  <tabstop>blockcheckbox</tabstop>
  <tabstop>previewtablecsv</tabstop>
  <tabstop>directioncombo</tabstop>
  <tabstop>prefixedit</tabstop>
  <tabstop>fitshdulist</tabstop>
  <tabstop>fitsdatasetname</tabstop>
  <tabstop>fitsdatacolumn</tabstop>
  <tabstop>fitssymcolumn</tabstop>
  <tabstop>fitsposcolumn</tabstop>
  <tabstop>fitsnegcolumn</tabstop>
  <tabstop>twod_previewedit</tabstop>
  <tabstop>twod_datasetsedit</tabstop>
  <tabstop>twod_xminedit</tabstop>
  <tabstop>twod_xmaxedit</tabstop>
  <tabstop>twod_yminedit</tabstop>
  <tabstop>twod_ymaxedit</tabstop>
  <tabstop>twod_invertrowscheck</tabstop>
  <tabstop>twod_invertcolscheck</tabstop>
  <tabstop>twod_transposecheck</tabstop>
  <tabstop>linkcheckbox</tabstop>
  <tabstop>importbutton</tabstop>
  <tabstop>closebutton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closebutton</sender>
   <signal>clicked()</signal>
   <receiver>importdialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>634</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel" >
     <x>318</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
